### DISCUSSION TITLE
Is it possible to install from Conda other than pip?

### DISCUSSION THREAD

**Discussion by yingbo** [pdm-project/pdm#1707] [Q&A] [answered]
Sometime, for ML projects, I need to use conda to make life easier. Just curious, when add a dependency, is it possible to let PDM use conda other than PIP? It is OK if everything is from conda but not pip.

I guess it's very hard, just wanna check out. Thanks!

**Comment by pawamoy** [comment]
PDM does not use pip under the hood anymore. And it supports PyPI-like indexes only, so no Conda forge(s) I'm afraid.

**Comment by frostming** [comment, marked as answer]
You could try [pdm-conda] maintained by the community

[pdm-conda]: https://github.com/macro128/pdm-conda

**Reply by yingbo** [reply]
Thanks! Looks fit my need. I'll take a closer look on the pdm-conda.
